ingester/ingester.go

```diff
@@ -286,3 +298,35 @@ func (fi *FirehoseIngester) handleRecordCreate(
 
 	return nil
 }
+
+func (fi *FirehoseIngester) handleRecordDelete(
```

Comment:
In `handleRecordCreate` we limit handling events to those created by actors that are present in the `CandidateActorCache`. 

I can see an argument for using the same cache as used for creates to filter out delete events. The reasoning is two fold:
- Avoid slowing down the ingestion of event creates/deletes. Checking the in-memory cache will be significantly faster than calling out to the DB - and we have a limited number of workers that are handling incoming events.
- Avoid loading the DB with `UPDATE`s for records we know most likely won't exist (there shouldn't be any likes/posts in the DB belonging to actors who aren't in the cache). This is relatively insignificant since I expect an UPDATE limited with a WHERE against the primary key will be relatively low load.

I suppose to play the devil's advocate, the cache could be incorrect/stale - however - I don't think this is a major concern. Since we'll only ever have a single ingester process, we know that the part handling the ingestion of create operations is using the same in-memory cache as the part handling the ingestion of delete operations. I'm also not **currently** worried about 100% correctness, I'm planning to either:
- Start dropping all events older than a month from the DB
- Build a reconciliation process that runs against actors in the DB and fetches all of their likes/posts from bsky and updates the DB accordingly.

Curious on your thoughts.

Reply:
Hmm, I originally checked the actor cache but (probably because I only just added my user via the cli) it couldn’t find my actor. That’s why I didn’t add the check.

If you’re not worried about 100% correctness, however, I’m more than happy to go with checking the actor cache! And then you can either prune old records every so often or sync with Bluesky, agreed.

Reply:
Yeah - with the feeds it feels like a few dropped/undeleted posts/likes don't significantly affect anything with the scale of the number of events we have now and I'd rather optimise for performance at ingestion point. Thanks for making this change.

And yeah, atm, the CAC takes at most a minute to update.